internal/service/cloudbackupschedule/data_source_cloud_backup_schedule.go

```diff
@@ -47,6 +56,11 @@ func DataSource() *schema.Resource {
 							Computed: true,
 						},
 						"replication_spec_id": {
+							Type:       schema.TypeString,
+							Computed:   true,
+							Deprecated: DeprecationMsgOldSchema,
```

Comment:
what's the effect of declaring a deprecated attribute in a DataSource?  is the warning message shown somewhere?

Reply:
will be shown if this is used as an output or used in an other resource as the value of an attribute:
```
output "output_var" {
  value = data.mongodbatlas_cluster.example_cluster.cluster_id
}

resource "mongodbatlas_<resource>" {
  example_attribute = data.mongodbatlas_cluster.example_cluster.cluster_id
}
```